package/MDAnalysis/topology/TOPParser.py

```diff
@@ -60,6 +60,7 @@
 
 TODO:
   Add support for Chamber-style topologies
+  Add support for atomnum topology attributes
```

Comment:
Is this atomic number style attributes? Not really relevant here but we might want to avoid calling them `atomnum` to a) avoid confusion with `resnum` and b) avoid confusion with whatever Desmond calls `atomnum`

Reply:
I think I might have misread https://github.com/MDAnalysis/mdanalysis/issues/2362#issue-505245572, I was under the impression that `atomnum` was our topology attribute for atomic numbers. As you mentioned in https://github.com/MDAnalysis/mdanalysis/issues/2651#issuecomment-605711880 it probably would be an idea to have an atomic number attribute. I can probably just change this to "Add support for storing atomic numbers", and we can decide what we should call this topology attribute at a later date?

Reply:
Sounds good! We have an `atomnum`, but [it's only used in the Desmond parser and is all 0s](https://github.com/MDAnalysis/mdanalysis/issues/2362); not sure if that's a bug or real information. 

```python
>>> from MDAnalysis.tests.datafiles import DMS
>>> dms = mda.Universe(DMS)
>>> dms.atoms.names
array(['N', 'HT1', 'HT2', ..., 'C', 'OT1', 'OT2'], dtype=object)
>>> dms.atoms.atomnums
array([0, 0, 0, ..., 0, 0, 0], dtype=int32)
```

Reply:
Hm, the desmond thing is a sqlite database if I remember right, so not sure how we're not getting that right.  I'd sooner coerce atomic numbers into elements for some uniformity rather than literalism in what the file provided.

Reply:
>  I'd sooner coerce atomic numbers into elements for some uniformity rather than literalism in what the file provided.

So my thought there was that as @lilyminium mentioned https://github.com/MDAnalysis/mdanalysis/issues/2651#issuecomment-605711880, whilst we have to trust that the information provided in topology files is accurate, there might be some use cases where (i.e. dummy atoms) where one wants to have a copy of the atomic number record to do some kind of further processing. Additionally, I think @cbouy mentioned that it might be good to have this available for RDKIT parsing. So it might be a good idea to have both attributes available.

That being said, there's an argument that having both is redundant and maybe it would be easier to assume atomic number -> element conversion is a "guess" and drop populating elements in the TOPParser (I think at the moment, it's the only one that does this kind of automatic conversion)?

My suggested way forward here would be that we pin this PR until after v1.0 is out, then we go ahead with this PR (my view is that if anything not guessing from atom names here is going to reduce user errors), and then we have a longer discussion on atomic numbers vs elements as a separate issue aimed at v2.0? This way if we happen to change our mind, it won't be reflected too much on users.